src/NLogShipper/NLogShipper.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">netstandard2.0;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">netcoreapp3.1;netstandard2.0;netstandard1.3</TargetFrameworks>
```

Comment:
Any reason for adding `netcoreapp3.1` since `netstandard2.0` is fully compatible?